orca-sql/src/test/groovy/com/netflix/spinnaker/orca/sql/pipeline/persistence/SqlExecutionRepositorySpec.groovy

```diff
@@ -71,17 +72,21 @@ class SqlExecutionRepositorySpec extends ExecutionRepositoryTck<ExecutionReposit
 
   @Override
   ExecutionRepository createExecutionRepository() {
-    com.netflix.spinnaker.kork.telemetry.InstrumentedProxy.proxy(
-      new DefaultRegistry(),
-      new SqlExecutionRepository("test", currentDatabase.context, mapper, new RetryProperties(), 10, 100, "poolName"),
-      "namespace")
+    return createExecutionRepository("test")
   }
 
   @Override
   ExecutionRepository createExecutionRepositoryPrevious() {
     new SqlExecutionRepository("test", previousDatabase.context, mapper, new RetryProperties(), 10, 100, "poolName")
   }
 
+  ExecutionRepository createExecutionRepository(String partition) {
```

Comment:
nit, I think a default argument would work nicely here 
```suggestion
  ExecutionRepository createExecutionRepository(String partition = "test") {
```

Reply:
can't because there is one already in the baseclass :(